docs/docs/setting-up-master-ui/configuring-master/configuring-master.mdx

```
@@ -1 +1,368 @@
+import Tabs from '@theme/Tabs';
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)